driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -165,6 +165,19 @@ public static Bson out(final String collectionName) {
         return new BsonDocument("$out", new BsonString(collectionName));
     }
 
+    /**
+     * Creates a $sample pipeline stage with the specified sample size
+     *
+     * @param size the sample size
+     * @return the $sample pipeline stage
+     * @mongodb.driver.manual reference/operator/aggregation/sample/  $sample
+     * @mongodb.server.release 3.2
+     * @since 3.2
+     */
+    public static Bson sample(final int size) {
+        return new BsonDocument("$sample", new BsonDocument("size", new BsonInt32(size)));
```

Comment:
You'll need to also add an option to allow disk use when this option is added.  I'm not sure if the API supports automatically adding it, though.  At the least, the docs should mention the potential need for this option.